src/backend/src/user_profile.rs

```diff
@@ -3,9 +3,24 @@ use ic_cdk::api::time;
 use ic_stable_structures::StableBTreeMap;
 use shared::types::{
     user_profile::{AddUserCredentialError, GetUserProfileError, StoredUserProfile},
-    CredentialType, Version,
+    CredentialType, Timestamp, Version,
 };
 
+fn store_user(
```

Comment:
`store_profile` or `store_user_profile` for consistency?

Reply:
done